### PR TITLE
Ales qgc camera settings grid

### PR DESCRIPTION
**Pull request by Ubaydulloh99**
Camera Settings Grid component 

image

### CODE REVIEW

File: qgcimages.qrc [hunk 1]
```diff
@@ -196,5 +196,8 @@
         <file alias="Yield.svg">src/ui/toolbar/Images/Yield.svg</file>
         <file alias="ZoomMinus.svg">src/FlightMap/Images/ZoomMinus.svg</file>
         <file alias="ZoomPlus.svg">src/FlightMap/Images/ZoomPlus.svg</file>
+        <file alias="Grid.svg">custom/res/Images/Grid.svg</file>
+        <file alias="grid+line.svg">custom/res/Images/grid+line.svg</file>
```
Comment (thanhkaist) [inline qgcimages.qrc]: grid+line.svg Change the file name. It should not use + in the file name. Suggestion: GridLine.svg

**Review by thanhkaist — COMMENTED** [review]
Make image name consistent

File: qgcimages.qrc (hunk 1 above)
Comment (Ubaydulloh99) [reply qgcimages.qrc]: ok

Comment (Ubaydulloh99) [reply qgcimages.qrc]: done

File: src/QmlControls/Grid.qml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                text: 'None'
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2
+                color: white
+                anchors.verticalCenter: parent.verticalCenter
+            }
+        }
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/Grid.qml]: Hardcoding. Please use property binding

File: src/QmlControls/Grid.qml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                color: white
+                text: qsTr("Grid")
+                font.pixelSize: ScreenTools.mediumFontPointSize * 2
+                anchors.verticalCenter: parent.verticalCenter
+            }
+        }
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/Grid.qml]: Hardcoding. Please use property binding

File: src/QmlControls/Grid.qml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+        anchors.topMargin: 20 // Editable margin between text1 and column1
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.bottom: parent.bottom
+
+        spacing: 5
+
+        ItemDelegate {
+            width: parent.width
+            height: 52
```
Comment (thanhkaist) [inline src/QmlControls/Grid.qml]: Hardcoding. Please use property binding

**Review by thanhkaist — APPROVED** [review]

**Pull request merged**